device-types/Quantum/QXS-484.yaml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Quantum
+model: QXS-484
+slug: quantum-qxs-484
+u_height: 5
+is_full_depth: true
+airflow: front-to-rear
+subdevice_role: parent
+power-ports:
```

Comment:
As this device as hot swappable PSUs these should be modeled as `module-bays` then create the required PSUs as modules if they don't already exist.

Reply:
Yes